### DISCUSSION TITLE
How is it possible to send  input to the react native packager process?

### DISCUSSION THREAD

**Discussion by balmasri** [microsoft/vscode-react-native#2108] [Q&A] [answered]
Upon starting the react native packager from the react native tools extension, we don't have a handle on the process and we are unable to use buttons like "r" or "d" to reload or show dev menu. Instead we have commands from the command palette that allow these functionalities.
How do these commands work? how is possible to send those reload or show dev menu commands to the metro bundler after the process is launched? is there a way to do that from a different vscode extension?

**Comment by EzioLi01** [comment, marked as answer]
I suggest that you can use react-native native command `npx react-native start` to start metro and launch app. Just using attach request configuration to attach debugger to your app.
